ibis/tests/test_impala_e2e.py

```diff
@@ -903,6 +903,179 @@ def test_sql_query_limits(self):
             assert table.count().execute() == 25
             assert table.count().execute(limit=10) == 25
 
+#  UDF Testing
+    def test_boolean_wrapping(self):
```

Comment:
Can you use a mix of array expressions and literal (scalar) values in these test cases? There is still no test case containing a non-literal expression (or any array expression, for that matter) as far as I can see